frontend/src/pages/login/components/LoginForm.tsx

```diff
@@ -0,0 +1,88 @@
+import { useCallback } from 'react'
+import { useForm } from 'react-hook-form'
+import { FormControl, Stack, useBreakpointValue } from '@chakra-ui/react'
+import isEmail from 'validator/lib/isEmail'
+
+import { FORM_GUIDE } from '~constants/externalLinks'
+import Button from '~components/Button'
+import FormErrorMessage from '~components/FormControl/FormErrorMessage'
+import FormLabel from '~components/FormControl/FormLabel'
+import Input from '~components/Input'
+import Link from '~components/Link'
+
+export type LoginFormInputs = {
+  email: string
+}
+
+interface LoginFormProps {
+  onSubmit: (inputs: LoginFormInputs) => Promise<void>
+}
+
+export const LoginForm = ({ onSubmit }: LoginFormProps): JSX.Element => {
+  const { handleSubmit, register, formState, setError } =
+    useForm<LoginFormInputs>()
+
+  const validateEmail = useCallback((value: string) => {
+    const isValidEmail = isEmail(value)
+    if (!isValidEmail) {
+      return 'Please enter a valid email'
+    }
+
+    const isGovDomain = value.split('@').pop()?.includes('gov.sg')
+    return isGovDomain || 'Please log in with a gov.sg email address.'
+  }, [])
+
+  const onSubmitForm = async (inputs: LoginFormInputs) => {
+    return onSubmit(inputs).catch((e) => {
+      setError('email', { type: 'server', message: e.message })
+    })
+  }
+
+  const isMobile = useBreakpointValue({ base: true, xs: true, lg: false })
```

Comment:
the behaviour of this hook is not clear to me from the docs - does `md` default to `true` or `false` in this case?

Reply:
`md` will follow `base`, so `true`. But I've removed this in favour of just hiding the display